packages/forms/src/components/FloatingSearch/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { useRef } from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Input, ButtonIcon, RenderIf } from 'react-rainbow-components';
+import { MagnifyingGlass, Close } from '@rainbow-modules/icons';
+import InternalOverlay from 'react-rainbow-components/components/InternalOverlay';
+import { StyledContainer, ButtonsContainer, StyledDivider, StyledClearButton } from './styled';
+import clearMessages from './clearMessages';
+
+const FloatingSearch = (props) => {
+    const { isOpen, triggerRef, onChange, placeholder, style, value, closeSearch } = props;
+    const inputRef = useRef(null);
```

Comment:
there is no need to init its value with null since it will init it already with null